values.yaml

```diff
@@ -16,7 +16,6 @@ openshift-pipelines:
 acs:
   central-endpoint: ${ACS__CENTRAL_ENDPOINT}
   api-token: ${ACS__API_TOKEN}
-developer-hub:
```

Comment:
Why is that line being removed?

Reply:
Ouch 🫣 Thanks for catching this up, I'm reverting this line.